apps/server/src/apps/h5p-editor.app.ts

```diff
@@ -24,6 +25,8 @@ async function bootstrap() {
 	// WinstonLogger
 	nestApp.useLogger(await nestApp.resolve(LegacyLogger));
 
+	nestApp.use(createAppLoggerMiddleware(await nestApp.resolve(Logger)));
```

Comment:
createAppLoggerMiddleware ist erstmal sehr allgemein benannt, hier wird allerdings ganz konkret ein request logger eingehangen auf Basis des Express Unterbaus. 
Daher wäre es sinnvoll die Funktion auch konkret zu benennen. 

Die zweite Frage wäre ob man das nicht mit den nest Observern / Interceptoren oder was auch immer abwickelt. Nur mal als Frage rein geworfen. Es kann aber durchaus sinnvoll sein, das bewusst aus dem Nest Code zu halten und nur im bootstrap zu haben.

Reply:
goal is simple solution on express level to not mangle with nestjs

Reply:
function renamed